src/components/ground-nav/ground-nav.scss

```diff
@@ -182,9 +183,11 @@ $_ground-nav-border-color: color.$base-gray-light;
   }
 
   .c-ground-nav__social {
+    border: 0 $_ground-nav-border-style $_ground-nav-border-color-fallback;
+    border-color: $_ground-nav-border-color;
```

Comment:
Again, I'm a fan of just setting border styles initially and then toggling width later. There's something about conditionally applying and un-applying border styles that just sticks in my craw.